spec/rspec/rails/configuration_spec.rb

```diff
@@ -256,6 +256,31 @@ def self.application; end
       expect(group.mailer_class).to be(a_mailer_class)
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
+
+    describe 'cleans test mailbox between each example in all rspec-rails example' do
```

Comment:
Not sure about this type of test in this file but I didn't wanted a feature test that will be visible in relish doc. Maybe you have a better idea?

Reply:
👍